File: PRAKTIKUM-PEMROGRAMAN1/MODUL5/C/PRAK505-2410817210024-MuhammadIbnuSina.c
#include <stdio.h>

void Biodata(int lahir, char nama[], char asal[]){
    int tahun_sekarang = 2020;
    int umur = tahun_sekarang-lahir;

    printf("Perkenalkan Nama Saya : %s\n",nama);
    printf("Umur Saya : %d\n",umur);
    printf("Saya Adalah Angkatan : %d\n",tahun_sekarang);
    printf("Asal Saya dari : %s\n",asal);
}

int main(){
    int tahunLahir;
    char A[20], B[15];
    scanf("%d", &tahunLahir);
    scanf(" %[^\n]%*c", &A);
    scanf(" %[^\n]%*c", &B);
    Biodata(tahunLahir,A,B);
    return 0;
}

File: PRAKTIKUM-PEMROGRAMAN1/MODUL5/Python/PRAK505-2410817210024-MuhammadIbnuSina.py
def Biodata(lahir, nama, asal):
    tahun_sekarang = 2020
    umur = tahun_sekarang-lahir

    print(f"Perkenalkan Nama Saya : {nama}")
    print(f"Umur Saya : {umur}")
    print(f"Saya Adalah Angkatan : {tahun_sekarang}")
    print(f"Asal Saya dari : {asal}")

tahunLahir = int(input())
A = input()
B = input()
Biodata(tahunLahir,A,B)